src/components/ChannelsAvailabilityMenuContent/ChannelsAvailabilityMenuContent.tsx

```diff
@@ -0,0 +1,53 @@
+import { Typography } from "@material-ui/core";
+import HorizontalSpacer from "@saleor/apps/components/HorizontalSpacer";
+import { Pill, PillColor } from "@saleor/macaw-ui";
+import ScrollableContent from "@saleor/plugins/components/PluginsList/PluginAvailabilityStatusPopup/ScrollableContent";
+import React from "react";
+import { MessageDescriptor } from "react-intl";
+import { useIntl } from "react-intl";
+
+import { messages } from "../ChannelsAvailabilityDropdown/messages";
+import { Channels } from "../ChannelsAvailabilityDropdown/utils";
+import { useStyles } from "./styles";
+
+export interface ChannelsAvailabilityMenuContentProps {
+  channels: Channels[];
+  labelFunction: (channelData: unknown) => MessageDescriptor;
```

Comment:
is `uknown` on purpose here?

Reply:
Yes. The idea is that `labelFunction` and `colorFunction` can be any functions, what's important is their return type -`MessageDescriptor` and `PillColor` respectively, so now when I think about it there can be more than one parameter. So it could be something like that:
`labelFunction: (...channelData: unknown[]) => MessageDescriptor`